src/styles/blocks.scss/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  width: 166px;
+  max-height: 16px;
+  margin: 0;
+  padding-top: 17px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../../images/star.svg);
+    background-repeat: no-repeat;
+    background-size: 80%;
+    align-self: flex-end;
+  }
+  &__star.active {
+    background-image: url(../../images/star-active.svg);
+  }
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__reviews {
```

Comment:
it shouldn't be a part of `stars` block - it's a part of `card` block. `stars` is only images (active or not, depends on block modificator like `--4`)